hibernate5/src/test/java/com/baeldung/hibernate/MultiTenantDaoHibernateIntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.hibernate;
+
+import java.io.IOException;
+
+import org.hibernate.SessionFactory;
+import org.junit.Test;
+
+import com.baeldung.hibernate.dao.SupplierDao;
+import com.baeldung.hibernate.pojo.Supplier;
+
+import static org.junit.Assert.assertNull;;
+
+public class MultiTenantDaoHibernateIntegrationTest {
+    @Test
+    public void givenDBMode_whenFetchingSuppliersByName_thenChecking() throws UnsupportedTenancyException, IOException {
+        SessionFactory sessionFactory = HibernateMultiTenantUtil.getSessionFactory();
+        
+        SupplierDao myDb1Dao = new SupplierDao(sessionFactory, "mydb1");
+        Supplier db1SupplierName = myDb1Dao.findByName("John");
```

Comment:
Test fails. Both this findByName and the second return null, because you did not insert the data into the database as you did in the other test class. Recommend creating a variable for the name "John", using the variable in both calls to findByName, and doing an assertNotNull on the first one.